### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by PgMiku** [Lost-MSth/Arcaea-server#44] [Q&A]
Hello everyone, I'm new one in here. How this thing work? I've read README more than 3 times, also read from Issues and Discussions, but I can't run it. It still error.  Can someone help me?

Thank you.

**Comment by qwerfd2-wustl** [comment]
ngl the first time I found this I was lost as well. There might be... helpful pages... around this repo.

**Reply by PgMiku** [reply]
What's that mean?

**Reply by qwerfd2-wustl** [reply]
_It's behind you_

There are helpful tips within the repo, look everywhere, and remember that nothing on the internet can be truly deleted.

Oh, and if you are the guy behind Arc-Extend: Stop.

**Reply by PgMiku** [reply]
1. Idk him
2. It doesn't work with me. Can u help?

**Reply by innocentvoice** [reply]
Are you sure you don't know him? Your Github repo is sus

**Reply by PgMiku** [reply]
> Are you sure you don't know him? Your Github repo is sus

After I quit his mod, I forget him, so in this time I don't know him anymore.

**Reply by innocentvoice** [reply]
well then the repo as qwerfd2 has said has some useful tips, try to look at it as this should not be discussed

**Reply by qwerfd2-wustl** [reply]
his name is quite sus too, the arc-extend dude is obsessive with miku for some reason.

**Reply by TechnoHT** [reply]
> his name is quite sus too, the arc-extend dude is obsessive with miku for some reason.

He's not, I can guarantee with that.

**Reply by TechnoHT** [reply]
> 1. Idk him
> 2. It doesn't work with me. Can u help?

Specify where you are stucking or step to reproduce it

**Reply by PgMiku** [reply]
> > 1. Idk him
> > 2. It doesn't work with me. Can u help?
> 
> Specify where you are stucking or step to reproduce it

First, idk how to use Charles, even I set map remote...
Second, when I run game + server open + proxy, it still not working, always 404 or it connected to Official server
That's all.

**Reply by innocentvoice** [reply]
> > > 1. Idk him
> > > 2. It doesn't work with me. Can u help?
> > 
> > 
> > Specify where you are stucking or step to reproduce it
> 
> First, idk how to use Charles, even I set map remote... Second, when I run game + server open + proxy, it still not working, always 404 or it connected to Official server That's all.

which client do you use?

**Reply by PgMiku** [reply]
> > > > 1. Idk him
> > > > 2. It doesn't work with me. Can u help?
> > > 
> > > 
> > > Specify where you are stucking or step to reproduce it
> > 
> > 
> > First, idk how to use Charles, even I set map remote... Second, when I run game + server open + proxy, it still not working, always 404 or it connected to Official server That's all.
> 
> which client do you use?

Charles 4.6.2
Arcaea server v2.8.4 ( April fool's )

**Comment by TechnoHT** [comment]
-->If you get 404 it’s either your network is not working properly or server’s configuration is wrong, dig it up.

**Reply by innocentvoice** [reply]
yea true, i agree

**Reply by PgMiku** [reply]
Still not working, I have tried a lot, I'm thinking it's from proxy I setting in my phone...
